tests/phpunit/tests/template.php

```diff
@@ -1109,6 +1216,27 @@ public function assertTemplateHierarchy( $url, array $expected, $message = '' )
 		$this->assertSame( $expected, $hierarchy, $message );
 	}
 
+	/**
+	 * Export PHP array as string formatted for pasting into unit test case.
```

Comment:
a brief example would go a long way. I have a rough idea of what this is supposed to do, which I assume is to aid in debugging test failures, but I really can’t visualize it.